source/server/admin/stats_handler.h

```diff
@@ -41,14 +42,25 @@ class StatsHandler : public HandlerContextBase {
                                     Http::ResponseHeaderMap& response_headers,
                                     Buffer::Instance& response, AdminStream&);
   Http::Code prometheusStats(absl::string_view path_and_query, Buffer::Instance& response);
+  void prometheusStats(const StatsParams& params, Buffer::Instance& response);
   Http::Code handlerContention(absl::string_view path_and_query,
                                Http::ResponseHeaderMap& response_headers,
                                Buffer::Instance& response, AdminStream&);
-
   Admin::RequestPtr makeRequest(absl::string_view path, AdminStream& admin_stream);
-  static Admin::RequestPtr makeRequest(Stats::Store& stats, bool used_only, bool json,
-                                       Utility::HistogramBucketsMode histogram_buckets_mode,
-                                       const absl::optional<std::regex>& regex);
+  static Admin::RequestPtr makeRequest(Stats::Store& stats, const StatsParams& params);
+
+private:
+  class Context;
```

Comment:
What are these? Seems like they're not used.

Reply:
good catch - detritus from other work on these files that was forked off.